includes/ProductCategory/Categories.php

```diff
@@ -129,13 +129,26 @@ private function get_categories() {
         global $wpdb;
 
         // get all categories
+        $table  = $wpdb->prefix . 'terms';
+        $fields = 'terms.term_id, terms.name, tax.parent AS parent_id';
+        $join   = "INNER JOIN `{$wpdb->prefix}term_taxonomy` AS tax ON terms.term_id = tax.term_id";
+        $where  = " AND tax.taxonomy = 'product_cat'";
+
+        $wpml_exists      = function_exists( 'wpml_get_current_language' );
+        $current_language = apply_filters( 'wpml_current_language', null );
```

Comment:
use `wpml_get_current_language()` instead of apply_filters( 'wpml_current_language', null );